pkg/operator/ceph/csi/template/nfs/csi-nfsplugin-provisioner-dep.yaml

```diff
@@ -21,6 +21,44 @@ spec:
       priorityClassName: {{ .ProvisionerPriorityClassName }}
       {{ end }}
       containers:
+        - name: csi-snapshotter
```

Comment:
put this behind a flag like rbd and cephfs. We need to provide an option for users to disable it in some cases

Reply:
> put this behind a flag like rbd and cephfs. We need to provide an option for users to disable it in some cases

I thought snapshot is stable enough and has been around for long enough now to not have a flag around it now.

Reply:
As it is installed via separate CRD, not every cluster will have it installed, and not everyone wants snapshot functionality.